lib/modules/platform/azure/index.ts

```diff
@@ -461,6 +462,22 @@ async function getMergeStrategy(
   );
 }
 
+function mapMergeStrategy(
```

Comment:
please move to commonts or util.ts, whatever exists

Reply:
Done, moved to util.ts.